src/utils.ts

```diff
@@ -47,8 +47,10 @@ export function handleErrors(func: () => Promise<void>) {
             await func();
         }
         catch (err) {
-            if (err.message !== CANCEL) {
-                vscode.window.showErrorMessage(err.message);
+            if (err instanceof Error) {
```

Comment:
Any specific reason for this change? Seems fine, just wondering.

Reply:
I think this is required for the extension to build with a newer version of TypeScript? It was the only way I could get it to work anyway. I've not used TypeScript before though, so it might have been a problem on my end.

Reply:
Seems like there was a similar change in this pull request: https://github.com/JacquesLucke/blender_vscode/pull/147#discussion_r1354863561